### PR TITLE
fix on_scroll_move to obey scroll_distance

### PR DESCRIPTION
**Pull request by darkopevec**
the function did not correctly check for minimum distance to travel before performing a scroll

### CODE REVIEW

File: kivy/uix/scrollview.py [hunk 1]
```diff
@@ -758,10 +758,24 @@ def on_scroll_move(self, touch):
             self._touch = False
             return self.on_scroll_start(touch, False)
         ud = touch.ud[uid]
-        mode = ud['mode']
```
Comment (KeyWeeUsr) [inline kivy/uix/scrollview.py]: Why? You just put it on two more lines with `ud['mode']`.

Comment (darkopevec) [inline kivy/uix/scrollview.py]: Sorry, I have a custom renamed widget and I edited this part multiple times. Obviously I missed this.

**Comment by matham** [comment]
There are unrelated rpi changes that you pushed (maybe unintentionally?).

**Comment by darkopevec** [comment]
I am still learning to use git.. please see https://github.com/kivy/kivy/pull/4644

**Pull request closed without merging**